src/python/ensembl/io/genomio/seq_region/dump.py

```diff
@@ -239,7 +239,9 @@ def main() -> None:
     parser.add_argument_src_path(
         "--external_db_map", default=DEFAULT_MAP.resolve(), help="File with external_db mapping"
     )
+    parser.add_log_arguments()
     args = parser.parse_args()
+    init_logging_with_args(args)
```

Comment:
Do we want to make this the standard, even if there is only one log argument involved?

Reply:
Yeah, no need to think, less risk of error if we want to add a log_file